includes/internal/emails/class-email-sender.php

```diff
@@ -113,6 +123,14 @@ public function send_email( $email_name, $replacements ) {
 			'Content-Type: text/html; charset=UTF-8',
 		];
 
+		$settings = $this->settings->get_settings();
+
+		if ( ! empty( $settings['email_reply_to_address'] ) ) {
+			$reply_to_address = $settings['email_reply_to_address'];
+			$reply_to_name    = isset( $settings['email_reply_to_name'] ) ? $settings['email_reply_to_name'] : '';
+			$headers[]        = "Reply-To: $reply_to_name <$reply_to_address>";
+		}
+
```

Comment:
Nothing critical, just a suggestion. As we are starting to work with headers now, can we move fetching headers to another function and call that here as we have some other headers too? And probably add a hook too so that the headers can be modified from other places?

Reply:
👍

Reply:
extracted 347e108fc1807814906d8b9bea7ec9f42483cc78 (wrong commit message 😢 )